apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -253,7 +256,12 @@ handle_request('GetName', Req, _Context) ->
             {200, [], #{name      => Name,
```

Comment:
I am a bit concerned returning the name of the entry.

1. Looking at `aens_claim_tx:check` I understand that ns_claim_tx does not need to have name already in form as returned by `aens_utils:to_ascii`;
2. Looking at `aens_claim_tx:process` I understand the actual name is not stored in the state tree (neither as specified in ns_claim_tx nor as `aens_utils:to_ascii`);
  1a. This seems not in line with [the spec](https://github.com/aeternity/protocol/blob/ce2af0674a5f427ddb2d9d5b547a302eeeeb81bb/AENS.md#aens-entry).
3. I see at least function `aec_conductor:get_name_entry` that queries the state tree passing the name as argument, then adding it in the return value - giving the impression to the reader that the name in that form is part of the state tree, while it is not;
4. I see when HTTP client queries `GetName` it gets back the name as passed in the request - giving the impression that the name is registered in that form - while it is not the case (rather the (hashed) ascii-fied name).